profile/CSIP.xml

```diff
@@ -164,7 +164,7 @@
             <requirement ID="CSIP2" REQLEVEL="MUST" RELATEDMAT="VocabularyType" EXAMPLES="metsRootElementExample1 metsRootElementExample2 metsRootElementExample3">
                 <description>
                     <head>General content type</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">The @TYPE attribute must be used for identifying the general type of the package (genre). A vocabulary is used. The vocabulary is going to evolve under the care of the DILCIS Board as additional content information type specifications are developed.</p>
+                    <p xmlns="http://www.w3.org/1999/xhtml">The @TYPE attribute must be used for identifying the general wider description of the content information type of the package. A vocabulary is used. The vocabulary is going to evolve under the care of the DILCIS Board as additional content information type specifications are developed.</p>
```

Comment:
The @TYPE attribute must be used to provide a general description of the content information type of the package. A fixed vocabulary is used which will evolve under the curation of the DILCIS Board as additional content information type specifications are developed.

Reply:
Fixed